### PR TITLE
bump libs for WebVR API v1.0

### PR DESCRIPTION
**Pull request by cvan**
r? @dmarcos @ngokevin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "style-attr": "^1.0.1",
     "three-dev": "^0.74.0-beta8",
     "tween.js": "^15.0.0",
-    "webvr-polyfill": "borismus/webvr-polyfill#3f47796"
```
Comment (cvan) [inline package.json]: when this is merged upstream, we'll point back to boris'

File: lib/three.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ if (THREE.TextureLoader) {
 require('../node_modules/three-dev/examples/js/loaders/OBJLoader');  // THREE.OBJLoader
 require('../node_modules/three-dev/examples/js/loaders/ColladaLoader');  // THREE.ColladaLoader
 require('../lib/vendor/Raycaster');  // THREE.Raycaster
-require('../node_modules/three-dev/examples/js/controls/VRControls');  // THREE.VRControls
-require('../node_modules/three-dev/examples/js/effects/VREffect');  // THREE.VREffect
+require('../node_modules/webvr-libs/VRControls');  // THREE.VRControls
```
Comment (cvan) [inline lib/three.js]: when this is merged upstream and three-dev is bumped, we'll point back to @ngokevin's

**Comment by cvan** [comment]
@ngokevin @dmarcos can we merge?

**Comment by cvan** [comment]
I want to merge this, but it looks like Brandon's still got several bugs to address with webvr-polyfill and `THREE.VREffect`.

Using the latest Oculus Chromium build, scenes using `THREE.VREffect` do not render correctly in the Oculus headset. See my comment in Brandon's PR: https://github.com/mrdoob/three.js/pull/8221#issuecomment-190182520 In the headset, the image appears super enlarged. I tried the webvr_cubes three.js WebVR example (http://threejs.org/examples/#webvr_cubes) with Brandon's branch, and I couldn't get the scene to render correctly. I also tried hooking it up to Sechelt and A-Frame scenes - same issue. It might be worth adding a three.js WebVR scene to Brandon's webvr-samples repo. (I filed https://github.com/toji/webvr-samples/issues/13 to add a three.js test scene.)

 The changes to `THREE.VREffect` in the PR currently do not work in Firefox (nor in the deprecated Chromium WebVR builds, but I know that's not an issue). See my comment here: https://github.com/mrdoob/three.js/pull/8221#discussion_r54210502

The changes to `webvr-polyfill` in the PR currently do not work in Firefox (nor in the deprecated Chromium WebVR builds, but I know that's not an issue). See my comment here: https://github.com/borismus/webvr-polyfill/pull/33#issuecomment-189135227

Also, looks like Brandon's making some changes directly to webvr-polyfill in the [webvr-samples directory](https://github.com/toji/webvr-samples/commit/7ff7408).

**Comment by ngokevin** [comment]
Should we bother merging this for 0.1.x?

**Comment by cvan** [comment]
> Should we bother merging this for 0.1.x?

well, all A-Frame sites don't/won't work in the Chromium builds. so I think we should.

**Pull request closed without merging**